app/backend/src/couchers/db.py

```diff
@@ -283,3 +284,17 @@ def can_moderate_node(session, user_id, node_id):
     return _can_moderate_any_cluster(
         session, user_id, [cluster.id for _, _, _, cluster in get_node_parents_recursively(session, node_id)]
     )
+
+
+def all_blocked_or_blocking_users(user_id):
```

Comment:
Needs a better name, e.g. `blocked_users_for(user_id)`

Reply:
It's also blocking users (i.e. users blocking them). It can be relevant_user_blocks, but that's less informative.

Reply:
Or do you just want me to add for? all_blocked_or_blocking_users_for(user_id)